spec/workers/codewars_recorder_spec.rb

```diff
@@ -6,7 +6,9 @@
     let(:user) { create(:user, codewars_username:"nbhartiya") }
     let!(:codewars_count) {
       codewars_count = 0
-      codewars_completed = HTTParty.get("http://www.codewars.com/api/v1/users/nbhartiya/code-challenges/completed")['data']
+      codewars_url = "http://www.codewars.com/api/v1/users/nbhartiya/" \
+                     "code-challenges/completed"
+      codewars_completed = HTTParty.get(codewars_url)['data']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.